app/lib/actions/katello/content_view/publish.rb

```diff
@@ -82,6 +88,15 @@ def plan(content_view, description = "", options = {})
             plan_action(Katello::Foreman::ContentUpdate, library, content_view) unless options[:skip_promotion]
             plan_action(ContentView::ErrataMail, content_view, library) unless options[:skip_promotion]
 
+            if import_only
+              plan_action(::Actions::Pulp3::Orchestration::ContentViewVersion::CopyVersionUnitsToLibrary, version)
+              concurrence do
+                version.importable_repositories.pluck(:id).each do |id|
+                  plan_action(Katello::Repository::IndexContent, id: id)
+                end
+              end
+            end
```

Comment:
does this need to be here? or can it be above right after the import?  Or better yet, can it be part of the improt action?

Reply:
Can be done right after import. I just want it o happen concurrently.. I can reorder